src/rules/store/prefix-selectors-with-select.ts

```diff
@@ -17,18 +21,22 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
       description:
         'The selector should start with "select", for example "selectThing".',
       recommended: 'warn',
+      suggestion: true,
     },
     schema: [],
     messages: {
-      [messageId]: 'The selector should start with "select".',
+      [prefixSelectorsWithSelect]: 'The selector should start with "select".',
+      [prefixSelectorsWithSelectSuggest]:
+        'Prefix the selector with the word "select" => `{{name}}`.',
```

Comment:
Feel free to suggest a better one :)

Reply:
Nit, I think "word" can be left out

```suggestion
        'Prefix the selector with the "select" => `{{name}}`.',
```